datahub-web-react/src/app/entity/shared/EntityDropdown/NodeParentSelect.tsx

```diff
@@ -77,12 +77,9 @@ function NodeParentSelect(props: Props) {
                 autoFocus={props.autofocus}
             >
                 {nodeSearchResults?.map((result) => (
-                    <Select.Option key={result?.entity?.urn} value={result.entity.urn}>
-                        <SearchResultContainer>
-                            <ParentEntities parentEntities={getParentGlossary(result.entity, entityRegistry)} />
```

Comment:
hm why are we removing the parent entities here? is that a requirement? as far as i know we want to show parent entities in glossary search bars like this to give better context what node or term you're choosing in case there is more than one with the same name